pyproject.toml

```diff
@@ -513,7 +513,11 @@ exclude = [
 max_line_length = 130
 
 [tool.pytest]
-junit_family = "xunit1"
+asyncio_mode = "auto"
```

Comment:
This are the settings we run pytest on.
We have to pass those manually in the `pre-commit-template` because `pytest` runs on his on working directory. But let's keep this.

Reply:
both is true for the `coverage` settings